### ISSUE TITLE
1.18.2 Litematica easy place mode placing fake block and shows im flying on the server.

### ISSUE THREAD

**Issue by WilliamJiH** [maruohon/litematica#537]
Minecraft Version: 1.18.2
Malilib Version: malilib-fabric-1.18.2-0.12.1
Server: Fabric

I'm playing on a vanilla fabric server with fly banned only, no other anti-cheats. Every single time I'm trying to use litematica's easy place mode to build some blocks, either the block is not synced with the server, or it gave me an error saying I'm flying while I'm actually standing on a solid block that have synced with the server. I have also tried to place the block using easy place mode where I was 1 block away, and turned on EASY_PLACE_VANILLA_REACH, but the blocks are still not synced. Do I need to also install litematica on the server side? Or is there a fix for me on client side? Thank you.

**Comment by maruohon** [comment]
I have no idea why the fly check would get triggered, Litematica shouldn't touch the player position in any way. (Unless you are using a printer extension mod, then those might send at least player rotation packets maybe?)

For the ghost block issue, read here:
https://github.com/maruohon/litematica/issues/521#issuecomment-1103581423
https://github.com/maruohon/litematica/issues/511

TL;DR; If the server doesn't have CarpetExtra, then you must set the `easyPlaceProtocolVersion` option in Generic to `Slabs Only` (or `None`). If it does have CarpetExtra, then it must be version **1.4.69** or later, and have the `accurateBlockPlacement` rule enabled. Otherwise any block placed via the Easy Place mode that has a rotation value will just become a ghost block

**Comment by WilliamJiH** [comment]
Thank you, after install CarpetExtra everything worked!
